yb_migrate/cmd/importData.go

```diff
@@ -380,6 +379,14 @@ func checkPrimaryKey(tableName string) bool {
 		table = strings.Split(tableName, ".")[0]
 	}
 
+	sourceDBType := ExtractMetaInfo(exportDir).SourceDBType
+	if sourceDBType == ORACLE {
+		table = strings.ToLower(table)
+	}
+
```

Comment:
Can you please confirm the behaviour for MySQL as well?

Reply:
yes tested that, MySQL exports the `table_data.sql` file into lowercase, so this is not needed.